tvm-tac/src/main/kotlin/org/ton/tac/TacInstructionHandlers.kt

```diff
@@ -113,6 +257,97 @@ object TacHandlerRegistry {
             is TvmContDictCalldictLongInst,
             -> CallDictHandler
 
+            is TvmDictGetDictgetInst, is TvmDictGetDictgetrefInst,
+            is TvmDictGetDictugetInst, is TvmDictGetDictugetrefInst,
+            is TvmDictGetDictigetInst, is TvmDictGetDictigetrefInst,
+            is TvmDictMayberefDictgetoptrefInst, is TvmDictMayberefDictigetoptrefInst,
+            is TvmDictMayberefDictugetoptrefInst,
+
+            is TvmDictNextDictgetnextInst, is TvmDictNextDictgetnexteqInst, is TvmDictNextDictgetprevInst,
+            is TvmDictNextDictgetpreveqInst,
+            is TvmDictNextDictugetnextInst, is TvmDictNextDictugetnexteqInst, is TvmDictNextDictugetprevInst,
+            is TvmDictNextDictugetpreveqInst,
+            is TvmDictNextDictigetnextInst, is TvmDictNextDictigetnexteqInst, is TvmDictNextDictigetprevInst,
+            is TvmDictNextDictigetpreveqInst,
+
+            is TvmDictSpecialDictugetexecInst, is TvmDictSpecialDictugetexeczInst,
+            is TvmDictSpecialDictugetjmpInst, is TvmDictSpecialDictugetjmpzInst,
+            is TvmDictSpecialDictigetexecInst, is TvmDictSpecialDictigetexeczInst,
+            is TvmDictSpecialDictigetjmpInst, is TvmDictSpecialDictigetjmpzInst,
+
+            is TvmDictSubSubdictgetInst, is TvmDictSubSubdictugetInst, is TvmDictSubSubdictigetInst,
+            is TvmDictSubSubdictrpgetInst, is TvmDictSubSubdicturpgetInst, is TvmDictSubSubdictirpgetInst,
+
+            is TvmDictPrefixPfxdictgetInst, is TvmDictPrefixPfxdictgetqInst,
+            is TvmDictPrefixPfxdictgetjmpInst, is TvmDictPrefixPfxdictgetexecInst,
+            -> DictGetHandler
```

Comment:
Probably better to move listing dict instructions separately. Instead, here check for `is TvmDictInst` (or how it is called), and list specific dict instructions in a separate function.